packages/react-components/src/components/RichTextEditor/RichTextSendBox.tsx

```diff
@@ -487,6 +484,14 @@ export const RichTextSendBox = (props: RichTextSendBoxProps): JSX.Element => {
     /* @conditional-compile-remove(file-sharing-acs) */
     const errorMessage =
       uploadErrorMessage || /* @conditional-compile-remove(rich-text-editor-image-upload) */ imageUploadErrorMessage;
+
+    /* @conditional-compile-remove(rich-text-editor-image-upload) */
+    let errorBarType = SendBoxErrorBarType.error;
```

Comment:
Just wanted to confirm that Alex is okay with the change

Reply:
Yes confirmed with Alex about this behaviour, hence the change